spec/controllers/events_controller_spec.rb

```diff
@@ -105,7 +105,7 @@
       let(:params) { FactoryGirl.attributes_for(:event) }
 
       it 'throws a not authorized error' do
-        expect { subject }.to raise_error(Pundit::NotAuthorizedError)
+        expect(subject).to redirect_to forbidden_path
```

Comment:
I agree with Brendan's sentiment above. Furthermore... if we're creating a common case for all auth errors (redirecting to a 403) we could create a [shared example](https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/example-groups/shared-examples) and replace all of our `raise_error` tests with something like:

```ruby
context "when a user does not have permissions" do
  # setup bad permissions here
  include_examples "failed authorization"
end
```

I'd look at doing something like this if we're checking for `Pundit::NotAuthorizedError` in several tests.